### ISSUE TITLE
Need another version bump

### ISSUE THREAD

**Issue by gaow** [vatlab/jupyterlab-sos#49]
I just installed JupyterLab 2.0.0 but cannot get SoS to work with it.

```
$ jupyter labextension install transient-display-data
An error occured.
ValueError: This extension does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab              Extension      Package
>=2.0.0 <2.1.0          >=1.0.5 <2.0.0 @jupyterlab/application
>=2.0.0 <2.1.0          >=1.0.5 <2.0.0 @jupyterlab/apputils
>=2.0.0 <2.1.0          >=1.0.5 <2.0.0 @jupyterlab/console
>=2.0.0 <2.1.0          >=1.0.5 <2.0.0 @jupyterlab/notebook
>=5.0.0 <5.1.0          >=4.0.5 <5.0.0 @jupyterlab/services
See the log file for details:  /tmp/jupyterlab-debug-rw23l1ep.log
$ jupyter labextension install jupyterlab-sos
An error occured.
ValueError: This extension does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab              Extension      Package
>=2.0.0 <2.1.0          >=1.0.5 <2.0.0 @jupyterlab/application
>=2.0.0 <2.1.0          >=1.0.5 <2.0.0 @jupyterlab/apputils
>=2.0.0 <2.1.0          >=1.0.5 <2.0.0 @jupyterlab/console
>=2.0.0 <2.1.0          >=1.0.5 <2.0.0 @jupyterlab/notebook
>=5.0.0 <5.1.0          >=4.0.5 <5.0.0 @jupyterlab/services
See the log file for details:  /tmp/jupyterlab-debug-b8qvq3mq.log
```

**Comment by BoPeng** [comment]
2.0.0 is out? Let me have a look.

**Comment by BoPeng** [comment]
There are big API changes and it is not something that can be done tonight....

**Comment by gaow** [comment]
No rush because I've been using Jupyter Notebook IDE for now which works fine, but just want to know what are the challenges related to the version bump, and are we expecting such problem to happen frequently down the road?

**Comment by BoPeng** [comment]
1.x to 2.x is a major change regarding API. I have managed to fix all but one API change which I had to ask the Jlab developers. Hopefully I can get it fixed tonight.

**Comment by bioworkflows** [comment]
Done but not thoroughly tested. Please update and let me know if you notice any problem.

**Comment by gaow** [comment]
Thanks @BoPeng it is better than before but still has an issue: what if you open this notebook:

[20200214_Variance_Component_Epistasis.ipynb.txt](https://github.com/vatlab/jupyterlab-sos/files/4314659/20200214_Variance_Component_Epistasis.ipynb.txt)

you'll see some code blocks got lighlighted yellow for no reason. Some other code blocks do appear fine though. I'm not attaching a screenshot but hopefully you'll see it on your end too.

**Comment by bioworkflows** [comment]
I can see the problem, let me check.

**Comment by bioworkflows** [comment]
Fixed and released 0.6.1

**Comment by gaow** [comment]
Thank you @BoPeng it seems to work well at least on the notebook I'm working on which contains R, Python, bash and SoS cells.
